README.md

```diff
@@ -8,7 +8,32 @@
 About
 =====
 
-**The Battle for Wesnoth** is an Open Source, turn-based tactical strategy game with a high fantasy theme, featuring both singleplayer and online/hotseat multiplayer combat. Fight a desperate battle to reclaim the throne of Wesnoth, or take hand in any number of other adventures.
+**The Battle for Wesnoth** is an Open Source, turn-based tactical strategy game with a high fantasy theme, featuring both singleplayer and online/hotseat multiplayer combat. Fight a desperate battle to reclaim the throne of Wesnoth, search for the key to immortality, or take part in any number of other adventures. 
+
+Gallery
+-------
+
+![Screenshot_20230213_014035](https://user-images.githubusercontent.com/87458186/218564775-51fd00e4-417e-476e-83b3-eeaa7c538ce8.png)
+![Screenshot_20230213_014257](https://user-images.githubusercontent.com/87458186/218564861-a7361c7d-081c-49e4-a4e7-bf3302525986.png)
+![Screenshot_20230213_020340](https://user-images.githubusercontent.com/87458186/218564900-da341ba3-93f7-4491-8bfc-475776518ca2.png)
+
+Features
+--------
+
+* 17 different campaigns, each with their own tactical challenges, emotionally charged story, and multiple difficulty levels. 
```

Comment:
Is the "17" number currently accurate? It keeps changing with every release, and that's just the mainline campaigns... (also there's the question of whether to count stuff like the Tutorial, or multiplayer campaigns like WC...)

Reply:
For now, I guess. Might shrink or increment. Who knows